docs/configuration/overview.md

```diff
@@ -9,21 +9,15 @@ description: Learn how to configure Yazi.
 
 There are three configuration files for Yazi:
 
-- [`yazi.toml`](./yazi.md) - General configuration
-- [`keymap.toml`](./keymap.md) - Keybinds configuration
-- [`theme.toml`](./theme.md) - Color scheme configuration
+- [`yazi.toml`](./yazi.md) - General configuration.
+- [`keymap.toml`](./keymap.md) - Keybinds configuration.
+- [`theme.toml`](./theme.md) - Color scheme configuration.
 
-You can find the default configurations at: https://github.com/sxyazi/yazi/tree/main/yazi-config/preset.
+You can find the default configuration files at https://github.com/sxyazi/yazi/tree/main/yazi-config/preset.
 
-To override any of the defaults, begin by copying the appropriate file to following location:
+To override any of the defaults, begin by copying the appropriate file from the directory linked above to the Yazi configuration directory. On Unix/Unix-like systems, the configuration directory is `~/.config/yazi/`, and on Windows, the configuration directory is `C:\Users\USERNAME\AppData\Roaming\yazi\config\` (replacing `USERNAME` with your username).
```

Comment:
I'm not a fan of this, I usually prefer a shorter content per line, combined with a list to better explain it - it's not only neater but also clearer.

Reply:
I think the list is "neater" but breaks up the flow of the page. Lists aren't suitable for everything...

Reply:
This sentence structure also makes it easier to mention `YAZI_CONFIG_HOME` and `XDG_CONFIG_HOME`...

Reply:
I agree that this is easier to read, but it's not easy to lookup. When people come back to find the paths, the list would stand out more by placing Unix and Windows on two different lines. This is also the same for someone reading it for the first time, where they can instantly see the information they need - the configuration file paths for both platforms - without having to search through dense text/paragraphs.

This is similar to a dictionary; it's designed for reference, not for reading cover to cover. My goal in writing documentation is aligned with this approach - I aim for documents to be easily searchable by highlighting crucial content, allowing users to locate it effortlessly. It's not about crafting elaborate narratives or stories, which might be rich and engaging but aren't conducive to quick searches.

Reply:
That philosophy doesn't really align with how the configuration pages are written though (e.g. https://yazi-rs.github.io/docs/configuration/yazi)! I find these really difficult to parse and understand, so maybe we can work on those next ahah.

I guess we can do the list...

Reply:
How about a happy medium?

> To override any of the defaults, begin by copying the appropriate file from the directory linked above to the Yazi configuration directory.:
> 
> - On Unix/Unix-like systems, the configuration directory is `~/.config/yazi/`.
> - On Windows, the configuration directory is `C:\Users\USERNAME\AppData\Roaming\yazi\config\` (replacing `USERNAME` with your username).

Reply:
"configuration directory" appears 3 times, and I believe 2 of these are redundant:

![CleanShot 2024-01-11 at 00 36 29](https://github.com/yazi-rs/yazi-rs.github.io/assets/17523360/645b5c34-a511-4ff0-ada5-6c7471c8b36b)

Reply:
What about the following? Seems as simple as it can be. Not a list though... I think this is best though!

> To override any of the defaults, begin by copying the appropriate file from the directory linked above to either `~/.config/yazi/` on Unix/Unix-like systems or `C:\Users\USERNAME\AppData\Roaming\yazi\config\` on Windows.

Reply:
Okay okay fine I'll do this: 

> To override any of the defaults, begin by copying the appropriate file from the directory linked above to either:
> 
> - `~/.config/yazi/` on Unix/Unix-like systems.
> - `C:\Users\USERNAME\AppData\Roaming\yazi\config\` on Windows.
>